LayoutTests/svg/dom/SVGAnimationElement-begin-end-arguments-typechecks.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<script src="../../resources/js-test.js"></script>
+<svg>
+  <animate></animate>
+</svg>
+<script>
+description('Check that invalid values of arguments throw TypeError.');
+
+var animateElm = document.querySelector('animate');
+
+debug('SVGAnimationElement');
+
+debug('');
+debug('beginElementAt(float offset)');
+shouldNotThrow('animateElm.beginElementAt(0)');
+shouldThrow('animateElm.beginElementAt(NaN)');
+shouldThrow('animateElm.beginElementAt(Infinity)');
```

Comment:
Add this case here as well and get rid of the other test.
```
shouldThrow('animateElm.beginElementAt()');
````

Reply:
Done - added this and also in 'endElementAt' as well.